### ISSUE TITLE
GS plugin failed to open

### ISSUE THREAD

**Issue by escalade** [flathub/net.pcsx2.PCSX2#29]
Seems to be a regression, this was working beautifully not long ago. Tried removing all installed flats + .var/app/net.pcsx2.PCS, no luck. Can't boot to BIOS (no iso) either.

Hardware: NUC8i5BEK
Flatpak version: 1.7.2
Desktop: Sway

Happy to supply additional information, let me know what you need.

**Comment by Equivocal90** [comment]
I just experienced a similiar issue on a Fedora 32 setup and found your post trying to troubleshoot. I figured I'd share what fixed my issue.

PCSX2 would open normally and when starting to boot either the BIOS or an ISO it would open the Output window but, it would remain completely black. PCSX2 would freeze and need to be killed. I set the GS plugin to Null and it no longer froze and would play the audio for the PS2 BIOS.

While running `flatpak update` wouldn't update anything, PCSX2 was still on version 1.5.0/i386 even thought the repo had 1.6.0/x86_64. For me, The issue was resolved by uninstalling and reinstalling the flatpak which pulled in the latest version.
